src/Device/deviceLister.ts

```diff
@@ -43,42 +41,84 @@ export const wrapDeviceFromNrfdl = (device: NrfdlDevice): Device => ({
 export const wrapDevicesFromNrfdl = (devices: NrfdlDevice[]): Device[] =>
     devices.map(wrapDeviceFromNrfdl);
 
-/**
+const filterDeviceTraits = (lhs: DeviceTraits, rhs: DeviceTraits) => {
```

Comment:
Could you please rewrite this function, in order to provide a bit more understanding to the reader.